### PR TITLE
Update handling of sample logs and monitors

### PR DESCRIPTION
**Pull request by AndreiSavici**
Update handling of some sample logs, and monitors. No associated issue.

**To test:**
The systemtests should pass

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [x] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [x] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/LoadEXED.py [hunk 1]
```diff
@@ -38,16 +47,29 @@ def PyInit(self):
                                           extensions = ["xml"]),
                              doc="Instrument definition file. If no file is specified, the default idf is used.")
 
+        self.declareProperty('AngleOverride', -255.0,
+                             doc = "Rotation angle of the EXED magnet.\nThis should be read from the data file!\n"
```
Comment (reimundILL) [inline Framework/PythonInterface/plugins/algorithms/LoadEXED.py]: May be good to consider a Rotation angle unit? I guess the unit is degrees.

**Comment by reimundILL** [comment]
New algorithm properties `OutputMonitorWorkspace` and `AngleOverride` may require an update of the release notes and the documentation.

File: Framework/PythonInterface/plugins/algorithms/LoadEXED.py [hunk 2]
```diff
@@ -38,16 +47,29 @@ def PyInit(self):
                                           extensions = ["xml"]),
                              doc="Instrument definition file. If no file is specified, the default idf is used.")
 
+        self.declareProperty('AngleOverride', -255.0,
```
Comment (reimundILL) [inline Framework/PythonInterface/plugins/algorithms/LoadEXED.py]: Can this property (not added at the end) break some existing user scripts?

Comment (AndreiSavici) [reply Framework/PythonInterface/plugins/algorithms/LoadEXED.py]: This algorithm was prepared by the instrument scientist. He is fine with that..

Comment (reimundILL) [reply Framework/PythonInterface/plugins/algorithms/LoadEXED.py]: Ok, then it should not be a problem.

**Comment by reimundILL** [comment]
- code looks good in general and some improvements were made
- absence of unit tests ? -> consider writing them
- exception handling new and looks good in general
- documentation could be updated with regard to the angle treatment
- minor thing in summary and documentation: one workspace will be created, and optionally a second one containing the monitor data

Could not run the algorithm but the instrument scientist seems to be happy, no risk then to merge it.

**Comment by AndreiSavici** [comment]
There is a systemtest instead of unittest. It contains data that can be loaded.

**Comment by reimundILL** [comment]
I tested with V15_0000016544_S000_P01.raw:
- OutputMonitorWorkspace will be created (I thought it's optional)
- different `AngleOverride` options tested successfully
- `OutputMonitorWorkspace` works

**Review by vreimund — APPROVED** [review]

**Pull request merged**